File: app/src/main/java/ir/touristland/Models/HotelPrices/Response.java
package ir.touristland.Models.HotelPrices;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Response {

    @SerializedName("result")
    private List<ResultItem> result;

    public List<ResultItem> getResult() {
        return result;
    }

    public void setResult(List<ResultItem> result) {
        this.result = result;
    }

    @Override
    public String toString() {
        return
                "Response{" +
                        "result = '" + result + '\'' +
                        "}";
    }
}

File: app/src/main/java/ir/touristland/Models/HotelPrices/Response.kt
package ir.touristland.Models.HotelPrices

import com.google.gson.annotations.SerializedName

class Response {

    @SerializedName("result")
    var result: List<ResultItem>? = null

    override fun toString(): String {
        return "Response{" +
                "result = '" + result + '\''.toString() +
                "}"
    }
}